api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java

```diff
@@ -116,12 +145,28 @@ public DbUser linkRasLoginGovAccount(String authCode, String redirectUrl) {
       }
       // Fetch user info.
       userInfoResponse = rasOidcClient.fetchUserInfo(tokenResponse.getAccessToken());
-
     } catch (IOException e) {
       log.log(Level.WARNING, "Failed to link RAS account", e);
       throw new ServerErrorException("Failed to link RAS account", e);
     }
-    return userService.updateRasLinkLoginGovStatus(getLoginGovUsername(userInfoResponse));
+
+    // If eRA is not already linked, check response from RAS see if RAS contains eRA Linking
+    // information.
+    // TODO(RW-7108): Make eRA optional
```

Comment:
Isn't eRA already optional here? What is there left todo - do we not want to capture it if it's turned off for the institution?

Reply:
I am planning to add some logic like
```
insitutionService.getTierRequirement(user);
if(rt or ct Require eRA), then do the check. 
```
But after thinking it through, even institution turn if off, it's still fine and easy to check eRA and mark eRA complete? Then I don't need to worry about institution requirement here.

Reply:
Yeah - seems OK to me to get this anyways if it's available. Institutions might in a rare scenario change their requirements later, so this might save someone extra registration steps.